### PR TITLE
Add initial configuration for translation platform

### PR DESCRIPTION
**Pull request by translation-platform**
**Owners:** 

### Welcome to Shopify [Translation Platform](https://development.shopify.io/engineering/developing_at_Shopify/write-code/internationalization/translation_platform)
This PR contains the configuration files required to have your source (English) strings sent for translation, and receive translations back into your repository.

### Please review this PR before merging it!

- [ ] Review the configuration to familiarize yourself with the [configuration options](https://development.shopify.io/engineering/developing_at_Shopify/write-code/internationalization/translation_platform#Configuration_file) available.
- [ ] Make sure the specified components in `translation.yml` cover all your source files.
- [ ] Make sure the name of the component represents the audience of your project:
  - `buyer` for end users of Shopify stores
  - `merchant` where the users are merchants
  - `partner` for Shopify partners

Questions about this PR? Hop in the `#intl-services` Slack channel. We're happy to help!

@Shopify/translation-platform

### CODE REVIEW

**Comment by francinen** [comment]
https://github.com/Shopify/shopify_app/pull/617 introduces new English strings to the PR which should be translated into other supported locales.

**Comment by translation-platform** [comment]
:clock1: **This pull request has been open for some time now** :clock1:

It is important to merge it as soon as possible in order to avoid dealing with merge conflicts.

More importantly the content in here really wants to reach production so everyone can benefit from it.

:green_heart:

**Comment by translation-platform** [comment]
:clock1: **This pull request has been open for some time now** :clock1:

It is important to merge it as soon as possible in order to avoid dealing with merge conflicts.

More importantly the content in here really wants to reach production so everyone can benefit from it.

:green_heart:

**Comment by translation-platform** [comment]
:clock1: **This pull request has been open for some time now** :clock1:

It is important to merge it as soon as possible in order to avoid dealing with merge conflicts.

More importantly the content in here really wants to reach production so everyone can benefit from it.

:green_heart:

**Comment by translation-platform** [comment]
:clock1: **This pull request has been open for some time now** :clock1:

It is important to merge it as soon as possible in order to avoid dealing with merge conflicts.

More importantly the content in here really wants to reach production so everyone can benefit from it.

:green_heart:

File: translation.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+source_language: en
+target_languages: [de, es, fr, it, ja]
+components:
+  - name: 'partner'
```
Comment (ragalie) [inline translation.yml]: I feel like the _users_ are merchants, since all of the strings are merchant-facing.

**Comment by translation-platform** [comment]
:clock1: **This pull request has been open for some time now** :clock1:

It is important to merge it as soon as possible in order to avoid dealing with merge conflicts.

More importantly the content in here really wants to reach production so everyone can benefit from it.

:green_heart:

File: translation.yml (hunk 1 above)
Comment (francinen) [reply translation.yml]: Restored the initial state of this file 👍

**Comment by translation-platform** [comment]
:clock1: **This pull request has been open for some time now** :clock1:

It is important to merge it as soon as possible in order to avoid dealing with merge conflicts.

More importantly the content in here really wants to reach production so everyone can benefit from it.

:green_heart:

**Comment by translation-platform** [comment]
:clock1: **This pull request has been open for some time now** :clock1:

It is important to merge it as soon as possible in order to avoid dealing with merge conflicts.

More importantly the content in here really wants to reach production so everyone can benefit from it.

:green_heart:

**Pull request merged**